README.md

```diff
@@ -1,62 +1,83 @@
+🛠 Want to contribute to the project? View our [contributors guide](./CONTRIBUTING.md)
 
-📬 Want to receive the 3Box newsletter? [Sign up here](https://mailchi.mp/c671ca2b8093/3box)
+➡️ Have you integrated 3Box? [Add your project to 3Box Dapp Universe](./COMMUNITY-PROJECTS.md)
 
-➡️ Have you integrated 3Box? [Add your dapp to 3Box Dapp Universe!](./COMMUNITY-PROJECTS.md)
+<br>
 
-📝 Want to reach out about a potential integration? [Complete this form!](https://airtable.com/shrDYkQCnzlVUvHGe)
+# 3Box Overview
+The 3Box system primarily consists of the [3Box.js API](https://github.com/3box/3box-js), a distributed user data network, and the [3Box Profiles App](https://3box.io).
 
-🛠 Want to collaborate on code? [View the Guide to Contributing](./CONTRIBUTING.md)
+## 3Box.js API
+[`3box-js`](https://www.github.com/3box/3box-js) is a client-side JavaScript library and API that allows applications to integrate with the 3Box data network. The 3Box.js API allows developers to read, write, and delete public and private data associated with the user. This library can be used to get profile information about an address, set profile information about an address, and onboard users. The 3Box.js API is available via a REST API and a GraphQL endpoint.
 
-## Goals
-* Make it easy for users to share information publicly, while preserving privacy when desired
-* Improve the onboarding experience for decentralized applications by making it easy for developers to get information about Ethereum users
-* Improve Ethereum usability with distributed database infrastructure that works in production today
-* Provide a decentralized system, but pragmatically utilize centralized components to facilitate the transition
+[`3box-graphql`](https://github.com/3box/3box-js-graphql) is a GraphQL endpoint that allows developers to write more efficient 3Box.js getProfile() queries. Now developers can ask our API for specific common user profile fields instead of needing to return the entire profile. For example, this is useful when querying for name and image for hundreds or thousands of profiles at once.
 
-## 3Box DB
-3Box DB consists of two components: [`3box-js`](https://www.github.com/3box/3box-js) which is a client library, and [`3box-address-server`](https://www.github.com/3box/3box-address-server) which is a server. Together they are used to keep track of user data stored in [`orbit-db`](https://github.com/orbitdb/orbit-db) instances. Learn more about the [3Box architecture and how we do access control with encryption](./ARCHITECTURE.md).
+## 3Box Data Network
+The 3Box user data network consists of a few core components: the [pinning server](https://www.github.com/3box/3box-pinning-server), the [address server](https://www.github.com/3box/3box-address-server), and a [caching service](https://www.github.com/3box/3box-caching-service). Together these components are used to keep track of user data stored in [`orbit-db`](https://github.com/orbitdb/orbit-db) instances on IPFS. 
```

Comment:
The caching service doesn't have it's own repo (`https://www.github.com/3box/3box-caching-service` is linked here). It's built into the pinning-server